src/bun.js/bindings/c-bindings.cpp

```diff
@@ -880,3 +880,45 @@ extern "C" const char* BUN_DEFAULT_PATH_FOR_SPAWN = "C:\\Windows\\System32;C:\\W
 #else
 extern "C" const char* BUN_DEFAULT_PATH_FOR_SPAWN = "/usr/bin:/bin";
 #endif
+
+#if OS(DARWIN)
+
+#if CPU(ARM64)
+#define BLOB_HEADER_ALIGNMENT 16 * 1024
```

Comment:
@Jarred-Sumner Do you think all MachO files should have the same blob header alignment? I couldn't find references which mentioned different blob header alignment size for intel and apple-chip macs